### PR TITLE
Full audit: manage a server

### PR DESCRIPTION
**Pull request by DonNateR**

### CODE REVIEW

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-ab01983a03/how-to/)

**Comment by cyrichardson** [comment]
@DonNateR Suggested edits here: 
[managing-a-cloud-server_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2159545/managing-a-cloud-server_cyrEdits.docx)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-2731ccac73/how-to/)

File: content/cloud-servers/manage-a-cloud-server.md [hunk 1]
```diff
@@ -50,16 +50,14 @@ You can create an image of a server or rebuild a server from a previously saved
 
 You can perform the following essential management tasks without logging directly into your server:
 
-- Click **Emergency Console** to open Linux shell window or Windows command prompt window
+- Click **Emergency Console** to open Linux shell window or Windows® command prompt window
```
Comment (RackerKate) [inline content/cloud-servers/manage-a-cloud-server.md]: Suggest "to open the..."

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-cdafd278ce/how-to/). :bow:

**Pull request merged**